src/assets/server/rest/v1/schemas/auth/auth-check-eula.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
File not used I think: to remove?

Reply:
It is

AuthValidtion.ts
```ts
this._authCheckEula = JSON.parse(fs.readFileSync(`${global.appRoot}/assets/server/rest/v1/schemas/auth/auth-check-eula.json`, 'utf8'));
```

AuthService.ts
```ts
filteredRequest = AuthValidator.getInstance().validateAuthCheckEula(req.query);
```

Same for the others below

Reply:
Ah ok, it for JSon schema validator, nothing to do with the OAS in that case, right?

Reply:
Right